### ISSUE TITLE
Why QBlocker.app reads my cookies（Chrome）

### ISSUE THREAD

**Issue by Xinghanzzy** [steve228uk/QBlocker#34]
/Applications/QBlocker.app/Contents/MacOS/QBlocker
manipulated 
/Users/****/Library/Application Support/Google/Chrome/Default/Cookies

**Comment by vit-lebediev** [comment]
Yeah, it seems it was too good to be true: a small, free, open-source app, which will not try to steal my data.

**Comment by Darrenzzy** [comment]
LOL!

**Comment by renguer0** [comment]
So that's confirmed? No reply from devs?

Thanks for sharing that info, we need to keep our data safe from goverments and scammers.

**Comment by osleg** [comment]
disclaimer: I am not a developer of this app

I checked the source code and there's nothing malicious in it. App is not modifying neither reading anything of value.
Thus being said I'd like to notice that the release .dmg file downloaded not from github and we don't have any hash to check validity, hence my advice is if you wanna use this app - git clone it and build yourself, just to be sure.
